Directory.Build.props

```diff
@@ -20,6 +20,7 @@
         <DebugType Condition="'$(GITHUB_ACTIONS)' == 'true'">embedded</DebugType>
 
         <ContinuousIntegrationBuild Condition="'$(GITHUB_ACTIONS)' == 'true'">True</ContinuousIntegrationBuild>
+		<LangVersion>preview</LangVersion>
```

Comment:
Is this necessary?

Reply:
At the moment yes, to lock on a System.Threading.Lock instance you need to have LangVersion preview, which will probably not be a requirement once .NET 9.0 is officially released.